Composer/plugins/samples/assets/projects/CustomAction/customaction.dialog

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
I thought since we modified the built-in CSharp template (code + schema), we don't actually need this sample,  every sample can be used as customized action sample.

Reply:
see https://github.com/microsoft/BotFramework-Composer/issues/2824#issuecomment-621603697